### PR TITLE
Paul problem submit

### PR DESCRIPTION
**Pull request by paglenn**
problem submission
history viewing
to be done: add user info, possibly. depends on database structure.

### CODE REVIEW

**Review by brok3turtl3 — APPROVED** [review]
Great work!

**Review by sifulsidd — APPROVED** [review]
Beautiful job!

File: client/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.5",
     "axios": "^1.4.0",
+    "postcss-loader": "^7.3.3",
```
Comment (joie-zhang) [inline client/package.json]: let's move postcss-loader into the devDependencies

File: client/package.json [hunk 2]
```diff
@@ -23,9 +24,11 @@
     "@babel/core": "^7.22.10",
     "@babel/preset-env": "^7.22.10",
     "@babel/preset-react": "^7.22.5",
+    "autoprefixer": "^10.4.15",
     "babel-loader": "^9.1.3",
     "css-loader": "^6.8.1",
     "html-webpack-plugin": "^5.5.3",
+    "postcss": "^8.4.28",
```
Comment (joie-zhang) [inline client/package.json:28]: and could we move postcss to the dependencies?

File: client/package.json [hunk 3]
```diff
@@ -23,9 +24,11 @@
     "@babel/core": "^7.22.10",
     "@babel/preset-env": "^7.22.10",
     "@babel/preset-react": "^7.22.5",
+    "autoprefixer": "^10.4.15",
```
Comment (joie-zhang) [inline client/package.json:24]: Could we put this in the dependencies?

**Review by joie-zhang — APPROVED** [review]
The progress is insane! Great work, Paul & Eric! :D

**Review by Ericesposito — APPROVED** [review]
It's coming together!

**Pull request merged**